unconstrained_test.go

```diff
@@ -1503,16 +623,5 @@ func testMinimize(t *testing.T, tests []UnconstrainedTest, method Method) {
 				}
 			}
 		}
-
```

Comment:
It seems that there was a mistake in the GulfResearchAndDevelopment function. It is fixed in #70, so it should be reviewed and merged first. Sorry for the confusion.